<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
	<master auto="start"/>

	<node pkg="motion_detection_sensor_status_publisher" type="sensor_status.py" respawn="false" name="motionj_analysis_sensor_status_publisher" output="screen">
		<rosparam file="$(find motion_detection_sensor_status_publisher)/config/parameters.yaml" command="load" />
	</node>
</launch>
